docs/config/README.md

```diff
@@ -1379,4 +1402,10 @@ command = "echo foo"  # shows output of command
 files = ["foo"]       # can specify filters
 when = """ test "$HOME" == "$PWD" """
 prefix = " transcending "
+
+[custom.time]
+command = "time /T"
+files = ["*.pst"]
+prefix = "transcending "
+shell = "pwsh.exe -NoProfile -Command"
```

Comment:
```suggestion
shell = ["pwsh.exe", "-NoProfile", "-Command", "-"]
```

Reply:
Good one! I will wait for us to merge your PR 1237 first, apply that commit suggestion, resolve conflicts and we'll shoot for a merge then. 👍